sriovnet_switchdev.go

```diff
@@ -43,6 +43,34 @@ func parsePortName(physPortName string) (pfRepIndex int, vfRepIndex int, err err
 	return pfRepIndex, vfRepIndex, err
 }
 
+func isSwitchdev(netdevice string) bool {
+	swIDFile := filepath.Join(NetSysDir, netdevice, netdevPhysSwitchID)
```

Comment:
Please use this API in GetVfRepresentor as well.

Reply:
@paravmellanox  I will do it in a follow-up patch has it not related to GetUplinkRepresentor API